crates/gpui/src/platform/mac/text_system.rs

```diff
@@ -638,45 +628,6 @@ impl From<FontStyle> for FontkitStyle {
     }
 }
 
-// Some fonts may have no attributes despite `core_text` requiring them (and panicking).
-// This is the same version as `core_text` has without `expect` calls.
-mod lenient_font_attributes {
```

Comment:
IIRC we did need it to fix the panics (as remarked by the comments by @SomeoneToIgnore). Hence, I'd rather have this code stay as is.

Reply:
I see, so this is about retrieving the system's font names? If that's the case, then I can confirm that the implementations of `DirectWrite` and `CosmicText` are correct.